styles/snapchat-web/catppuccin.user.css

```diff
@@ -0,0 +1,116 @@
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    /* Snapchat applies the variables (overrides?) to specific elements so we need to apply to every element (with high specificity) to override those. */
+    &,
+    body > main#root * {
+      --sigMain: @base;
+      --sigSurface: @mantle;
+      --sigSurfaceRGB: red(@mantle) green(@mantle) blue(@mantle);
+      --sigAboveSurface: @crust;
+      --sigSurfaceDown: @base;
+      --sigSubscreen: @crust;
+      --sigTextField: @crust;
+      --sigDivider: @surface0;
+      --sigDividerLight: @surface0;
+      --sigBackgroundPrimary: @base;
+      --sigBackgroundSecondary: @mantle;
+      --sigBackgroundSecondaryHover: darken(@mantle, 5%);
+      --sigBackgroundTertiary: @crust;
+      --sigBackgroundMessageHover: lighten(@crust, 7%);
+      --sigOverlay: fade(@overlay0, 0.4);
+      --sigOverlayHover: fade(@overlay0, 0.35);
+
+      --sigTextPrimary: @text;
+      --sigButtonOnPrimary: @text;
+      --sigTextSecondary: @subtext0;
+      --sigButtonOnSecondary: @subtext0;
+      --sigTextTertiary: @subtext1;
+      --sigButtonOnTertiary: @subtext1;
+      --sigTextNegative: @red;
+      --sigButtonOnSuccess: @base;
+      --sigButtonOnError: @base;
+      --sigButtonOnNegative: @base;
+      --sigButtonOnChatSurfaceCalling: @base;
+
+      --sigButtonPrimary: @accent-color;
+      --sigButtonPrimaryHover: darken(@accent-color, 5%);
+      --sigButtonSecondary: @surface0;
+      --sigButtonSecondaryHover: @surface1;
+      --sigButtonNegative: @red;
+      --sigButtonError: @red;
+      --sigButtonErrorHover: darken(@red, 5%);
+      --sigButtonSuccess: @green;
+      --sigButtonSuccessHover: darken(@green, 5%);
+      --sigIconCalling: @green;
+      --sigIconCallingHover: darken(@green, 5%);
+      --sigChatSurfaceCalling: @green;
+      --sigChatSurfaceCallingDisabled: darken(@green, 20%);
+
+      --sigChat: @blue;
+      --sigChatIcon: @blue;
+      --sigSnapWithoutSound: @red;
+      --sigSnapWithSound: @mauve;
+    }
```

Comment:
```suggestion
    }
    
    path[fill="#fff"] {
      fill: @crust;
    }
```

This should fix the poor contrast on similar buttons to this one:
![image](https://github.com/catppuccin/userstyles/assets/71222764/843b8c0d-da7b-42a5-bb3b-462acd083e44)